mono/mini/mini-llvm.c

```diff
@@ -3154,7 +3191,15 @@ emit_init_method (EmitContext *ctx)
 	 * This enables llvm to keep arguments in their original registers/
 	 * scratch registers, since the call will not clobber them.
 	 */
-	set_call_preserveall_cc (call);
+
+	// FIXME? Using this with mono debug info causes unwind.c to explode when
+	// parsing some of these registers saved by this call. Can't unwind through it.
+	// Not an issue with llvmonly because it doesn't use that DWARF
```

Comment:
this comment seems to belong to `LLVMSetInstructionCallConv ()`. move it there. the comment above belongs to `set_call_preserveall_cc ()`